webapp/src/services/registration/registration.service.js

```diff
@@ -0,0 +1,102 @@
+import axios from "axios";
+import { authHeader } from "../base.service";
+
+const baseUrl = process.env.REACT_APP_API_URL;
+
+export const registrationService = {
+  getRegistrationForm,
+  getRegistrationResponse,
+  submitResponse,
+  getOffer
+};
+
+function getOffer(eventId) {
```

Comment:
Don't we have an offer service that does this?

Reply:
yeah, in `offer.service.js`